doc/user/content/sql/create-materialized-view.md

```diff
@@ -1,78 +1,92 @@
 ---
 title: "CREATE MATERIALIZED VIEW"
-description: "`CREATE MATERIALIZED VIEW` creates a materialized view, which Materialize will incrementally maintain as updates occur to the underlying data."
+description: "`CREATE MATERIALIZED VIEW` defines a view that is persisted in storage and incrementally updated as new data arrives."
```

Comment:
Is "storage" a term of art that we're introducing for the user-facing docs? Like, storage with a capital S? Or should we say something like "durable storage" to make it generic?

```suggestion
description: "`CREATE MATERIALIZED VIEW` defines a view that is persisted in durable storage and incrementally updated as new data arrives."
```

Also maybe "a view whose contents are persisted...", to draw the distinction between the view definition and the data of the view itself.

Reply:
> Is "storage" a term of art that we're introducing for the user-facing docs?

I was using storage and durable storage interchangeably, but it does make more sense to use "durable storage" through and through so it's clearer! We definitely don't want to introduce **S**torage.